### PR TITLE
Move VUMI_INSTALLED_APPS into a separate file so it's accessible to Twisted workers.

### PR DESCRIPTION
**Pull request by hodgestar**
This is a temporary solution until we have a more complete share configuration framework.

### CODE REVIEW

**Comment by jerith** [comment]
Looks reasonable. :+1: once Travis is happy.

File: go/base/utils.py [hunk 1]
```diff
@@ -187,7 +148,18 @@ def get_conversation_view_definition(conversation_type, conv=None):
...
     # Scoped import to avoid circular deps.
     from go.router.view_definition import RouterViewDefinitionBase
-    router_pkg = get_router_pkg(router_type, ['definition', 'view_definition'])
+    try:
+        router_pkg = get_router_pkg(
+            router_type, ['definition', 'view_definition'])
+    except UnknownRouterType:
+        # To handle obsolete routers that are still viewable
+        if router_type not in obsolete_router_types():
+            raise
```
Comment (justinvdm) [inline go/base/utils.py:157]: Shouldn't we be raising something here?

Comment (justinvdm) [inline go/base/utils.py:157]: Could we maybe add a test for this case, and a similar test for the case of obsolete conversation types?

Comment (hodgestar) [inline go/base/utils.py:157]: A "raise" by itself re-raises the current exception.

Comment (hodgestar) [inline go/base/utils.py:157]: Test was added. :)

Comment (justinvdm) [inline go/base/utils.py:157]: Ah, I see :) /me goes back to python school

**Comment by justinvdm** [comment]
Looks good. :+1: from me once Travis is happy.

**Comment by jerith** [comment]
Lots of `AttributeError: 'Settings' object has no attribute 'VUMI_INSTALLED_ROUTERS'` in the Travis builds, so :-1: until those are fixed.

**Comment by jerith** [comment]
I'm happy with this once we have a successful build.

**Pull request merged**